examples/react-gqty/src/components/SignIn.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react'
+
+import { useSignInEmailPasswordless } from '@nhost/react'
+
+export function SignIn() {
+  const [email, setEmail] = useState('')
+
+  const { signInEmailPasswordless, error } = useSignInEmailPasswordless()
+
+  const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault()
+    await signInEmailPasswordless(email)
+    alert('Magic Link Sent')
+  }
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit}>
+        <div>
+          <input type="email" placeholder="email" onChange={(e) => setEmail(e.target.value)} />
+        </div>
+        <div>
+          <button type="submit">Sign In</button>
```

Comment:
I would be more explicit:
```suggestion
          <button type="submit">Sign In with a magic link</button>
```